ouroboros-network-framework/src/Ouroboros/Network/InboundGovernor.hs

```diff
@@ -114,6 +116,10 @@ inboundGovernor tracer serverControlChannel inboundIdleTimeout
                         tracer
                         (igsCountersCache state)
                         (inboundGovernorCounters state)
+      traceWith tracer $ TrRemoteState $
+            mkRemoteSt . csRemoteState
+        <$> igsConnections state
+
```

Comment:
Should this be traced with `Cache`?

Reply:
This is a debug tracer, so it's better to not cache it.